tests/data/patches/polygons.snopatch

```diff
@@ -8,12 +8,8 @@
e&gt;&lt;/gmd:EX_GeographicBoundingBox&gt;&lt;/gmd:geographicElement&gt;&lt;/gmd:EX_Extent&gt;&lt;/gmd:extent&gt;&lt;/gmd:MD_DataIdentification&gt;&lt;/gmd:identificationInfo&gt;&lt;gmd:distributionInfo&gt;&lt;gmd:MD_Distribution&gt;&lt;gmd:transferOptions&gt;&lt;gmd:MD_DigitalTransferOptions&gt;&lt;gmd:onLine&gt;&lt;gmd:CI_OnlineResource&gt;&lt;gmd:linkage&gt;&lt;gmd:URL&gt;https://data.linz.govt.nz/layer/50836-nz-waca-adjustments/&lt;/gmd:URL&gt;&lt;/gmd:linkage&gt;&lt;/gmd:CI_OnlineResource&gt;&lt;/gmd:onLine&gt;&lt;/gmd:MD_DigitalTransferOptions&gt;&lt;/gmd:transferOptions&gt;&lt;/gmd:MD_Distribution&gt;&lt;/gmd:distributionInfo&gt;&lt;gmd:dataQualityInfo&gt;&lt;gmd:DQ_DataQuality&gt;&lt;gmd:scope&gt;&lt;gmd:DQ_Scope&gt;&lt;gmd:level&gt;&lt;gmd:MD_ScopeCode codeList=\"http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#MD_ScopeCode\" codeListValue=\"dataset\"&gt;dataset&lt;/gmd:MD_ScopeCode&gt;&lt;/gmd:level&gt;&lt;gmd:levelDescription&gt;&lt;gmd:MD_ScopeDescription&gt;&lt;gmd:other&gt;&lt;gco:CharacterString&gt;dataset&lt;/gco:CharacterString&gt;&lt;/gmd:other&gt;&lt;/gmd:MD_ScopeDescription&gt;&lt;/gmd:levelDescription&gt;&lt;/gmd:DQ_Scope&gt;&lt;/gmd:scope&gt;&lt;gmd:lineage&gt;&lt;gmd:LI_Lineage&gt;&lt;gmd:statement&gt;&lt;gco:CharacterString&gt;Adjustments areas are not available for data prior to the inception of Landonline (2001).&lt;/gco:CharacterString&gt;&lt;/gmd:statement&gt;&lt;/gmd:LI_Lineage&gt;&lt;/gmd:lineage&gt;&lt;/gmd:DQ_DataQuality&gt;&lt;/gmd:dataQualityInfo&gt;&lt;gmd:metadataConstraints&gt;&lt;gmd:MD_SecurityConstraints&gt;&lt;gmd:classification&gt;&lt;gmd:MD_ClassificationCode codeList=\"http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#MD_ClassificationCode\" codeListValue=\"unclassified\"&gt;unclassified&lt;/gmd:MD_ClassificationCode&gt;&lt;/gmd:classification&gt;&lt;/gmd:MD_SecurityConstraints&gt;&lt;/gmd:metadataConstraints&gt;&lt;gmd:metadataConstraints&gt;&lt;gmd:MD_LegalConstraints&gt;&lt;gmd:useLimitation&gt;&lt;gco:CharacterString&gt;Crown copyright reserved&lt;/gco:CharacterString&gt;&lt;/gmd:useLimitation&gt;&lt;gmd:useConstraints&gt;&lt;gmd:MD_RestrictionCode codeList=\"http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#MD_RestrictionCode\" codeListValue=\"copyright\"&gt;copyright&lt;/gmd:MD_RestrictionCode&gt;&lt;/gmd:useConstraints&gt;&lt;/gmd:MD_LegalConstraints&gt;&lt;/gmd:metadataConstraints&gt;&lt;gmd:metadataConstraints&gt;&lt;gmd:MD_LegalConstraints&gt;&lt;gmd:useLimitation&gt;&lt;gco:CharacterString&gt;Released under Creative Commons Attribution 4.0 International&lt;/gco:CharacterString&gt;&lt;/gmd:useLimitation&gt;&lt;gmd:useConstraints&gt;&lt;gmd:MD_RestrictionCode codeList=\"http://asdd.ga.gov.au/asdd/profileinfo/gmxCodelists.xml#MD_RestrictionCode\" codeListValue=\"license\"&gt;license&lt;/gmd:MD_RestrictionCode&gt;&lt;/gmd:useConstraints&gt;&lt;/gmd:MD_LegalConstraints&gt;&lt;/gmd:metadataConstraints&gt;&lt;/gmd:MD_Metadata&gt;</MDI>\n  </Metadata>\n</GDALMultiDomainMetadata>\n"
```

Comment:
This isn't right...

GDAL's "metadata" is a wrapper/container - in this case the metadata root should be the `<gmd:MD_Metadata>` element, and the `md_standard_uri` field in `gpkg_metadata` should be `http://www.isotc211.org/2005/gmd`

Reply:
This is just the metadata that is attached to the test data. As far as I know, it's been like this since you created the test data over a year ago. This particular test actually recreates the polygons test repo by applying a patch - so the patch contains all the same data as the polygons test repo.

I can try to unwrap the XML in the test dta manually if you want. I don't know why it's how it is, but I don't have any evidence that it hasn't been accurately preserved by Sno - kept exactly as it was supplied to Sno long ago.

Reply:
Fixed XML for polygons test repo in particular, in a separate commit.

Weird test data is unique to polygons test repo and doesn't seem to point to any particular Sno issue.